fili-core/src/main/java/com/yahoo/bard/webservice/web/JsonAndJsonApiResponseWriter.java

```diff
@@ -71,6 +78,35 @@ public void writeMetaObject(
 
         generator.writeObjectFieldStart("meta");
 
+        if (request instanceof DataApiRequest) {
+            DataApiRequest dataApiRequest = (DataApiRequest) request;
+            Set<LogicalMetric> logicalMetricSet = dataApiRequest.getLogicalMetrics();
+            generator.writeObjectFieldStart("schema");
+            for (LogicalMetric lm : logicalMetricSet) {
+                LogicalMetricInfo lmi = lm.getLogicalMetricInfo();
+                if (lmi != null) {
+                    generator.writeObjectFieldStart(lmi.getName());
+
+                    if (lmi.getType() != null) {
+                        generator.writeStringField("type", String.format("%1$s", lmi.getType().getType()));
+                        if (lmi.getType().getSubType() != null) {
+                            generator.writeStringField("subtype",
+                                    String.format("%1$s", lmi.getType().getSubType()));
+                        }
+                    }
+
+                    if (lmi.getType().getTypeMetadata() != null && lmi.getType().getTypeMetadata().size() != 0) {
+                        for (Map.Entry<String, String> entry : lmi.getType().getTypeMetadata().entrySet()) {
+                            generator.writeObjectField(entry.getKey(), entry.getValue());
```

Comment:
this looks a little off to me.  
The values of the map are themselves strings not objects.
so either we should be writing the whole map as an object with some name or we should be using writeStringField as the lines above do.